apiserver/facades/client/controller/controller.go

```diff
@@ -620,11 +587,11 @@ func (c *ControllerAPI) initiateOneMigration(ctx context.Context, spec params.Mi
 	if err != nil {
 		return "", errors.Trace(err)
 	}
-	applicationService, err := c.applicationServiceGetter(ctx, coremodel.UUID(hostedState.ModelUUID()))
+	applicationService, err := c.applicationServiceGetter(ctx, coremodel.UUID(modelTag.Id()))
```

Comment:
Now that we are not using `hostedState` here can we shift the creation of `hostedState` further down into the function closer to where it is first needed?

Reply:
Changed with thanks!